chain/types/bigint.go

```diff
@@ -124,6 +124,22 @@ func (bi *BigInt) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+var sizeUnits = []string{"B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB"}
```

Comment:
if we're just copying this from somewhere, can we reuse it?

Reply:
oh nvm, i see. we're moving it to here

Reply:
I moved this from `cli/utils.go`